src/effect.h

```diff
@@ -225,6 +225,8 @@ class effect : public JsonSerializer, public JsonDeserializer
         bool get_harmful_cough() const;
         /** Returns the percentage value by further applications of existing effects' duration is multiplied by. */
         int get_dur_add_perc() const;
+        /** Returns the duration interval at which the effects intensity is decremented */
```

Comment:
"Returns the number of turns it takes for the intensity to fall by 1 or 0 if intensity isn't based on duration.